hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java

```diff
@@ -748,6 +753,24 @@
     public static final HazelcastProperty QUERY_OPTIMIZER_TYPE
             = new HazelcastProperty("hazelcast.query.optimizer.type", QueryOptimizerFactory.Type.RULES.toString());
 
+    /**
+     * Type of Query Index result copying behavior.
+     * Valid Values:
+     * <ul>
+     * <li>READ - copy the results of an index query on index read. Should be used in index-write intensive cases.
+     * (default value)</li>
+     * <li>WRITE - copy the results of an index query on a write to an index. Should be used in index-read intensive
```

Comment:
Isn't the `WRITE` case more something like `internal data structures of the index are modified with copy-on-write semantics. Previously returned indexed query results reflect the state of the index at the time of the query and are not affected by future index modifications.` ? (also notice the double `</li>` closing element)

Reply:
ha, thx for writing it ;) Fixed